ui/src/app/components/render-resource-card.spec.tsx

```diff
@@ -53,4 +54,49 @@ describe('renderResourceCard', () => {
         expect(wrapper.exists()).toBeTruthy();
         expect(wrapper.text()).toBe('');
     });
+
+    it('renders a dataset menu', () => {
+      const testDataSet = {
+        ...stubResource,
+        dataSet: {
+          ...stubDataSet(),
+          prePackagedConceptSet: [PrePackagedConceptSetEnum.FITBIT]
+        }
+      } as WorkspaceResource;
+
+      const menu = renderResourceCard({
+        resource: testDataSet,
+        existingNameList: [],
+        onUpdate: async () => {},
+        menuOnly: true,
+      });
+      const wrapper = mount(menu);
+      wrapper.find({'data-test-id': 'resource-card-menu'}).first().simulate('click');
+      expect(wrapper.text()).toContain('Export to Notebook');
+      expect(wrapper.text()).not.toContain('Extract VCF Files');
+    });
+
+    it('renders a dataset menu, with WGS', () => {
+      const testDataSet = {
+        ...stubResource,
+        dataSet: {
+          ...stubDataSet(),
+          prePackagedConceptSet: [
+            PrePackagedConceptSetEnum.PERSON,
```

Comment:
does this `PERSON` concept set mean anything in the context of this test? as in, affects the behavior in some way.

Reply:
From a whitebox perspective, no. From a blackbox perspective, it's just ensuring we don't overfit to the scenario where WGS is the only element. PERSON is not specifically chosen, then just needs to be any other prepackaged concept set